x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/list_page/index.tsx

```diff
@@ -70,61 +162,116 @@ export const AgentConfigListPage: React.FunctionComponent<{}> = () => {
-        defaultMessage: 'Namespace',
-      }),
-      render: (namespace: string) => (namespace ? <EuiBadge>{namespace}</EuiBadge> : null),
-    },
-    {
-      field: 'description',
-      name: i18n.translate('xpack.ingestManager.agentConfigList.descriptionColumnTitle', {
-        defaultMessage: 'Description',
-      }),
-    },
-    {
-      field: 'datasources',
-      name: i18n.translate('xpack.ingestManager.agentConfigList.datasourcesCountColumnTitle', {
-        defaultMessage: 'Datasources',
-      }),
-      render: (datasources: AgentConfig['datasources']) => (datasources ? datasources.length : 0),
-    },
-    {
-      name: i18n.translate('xpack.ingestManager.agentConfigList.actionsColumnTitle', {
-        defaultMessage: 'Actions',
-      }),
-      actions: [
-        {
-          render: ({ id }: AgentConfig) => {
-            return (
-              <EuiLink href={`${DETAILS_URI}${id}`}>
+  const columns = useMemo((): Array<
+    EuiTableFieldDataColumnType<AgentConfig> | EuiTableActionsColumnType<AgentConfig>
+  > => {
+    const cols = [
+      {
+        field: 'name',
+        name: i18n.translate('xpack.ingestManager.agentConfigList.nameColumnTitle', {
+          defaultMessage: 'Name',
+        }),
+        // FIXME: use version once available - see: https://github.com/elastic/kibana/issues/56750
+        render: (name: string, agentConfig: AgentConfig) => (
+          <EuiFlexGroup gutterSize="s" wrap={true} alignItems="baseline">
+            <EuiFlexItem grow={false}>
+              <EuiLink href={`${DETAILS_URI}${agentConfig.id}`}>{name || agentConfig.id}</EuiLink>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiText color="subdued" size="xs">
                 <FormattedMessage
-                  id="xpack.ingestManager.agentConfigList.viewActionLinkText"
-                  defaultMessage="view"
+                  id="xpack.ingestManager.agentConfigList.revisionNumber"
+                  defaultMessage="rev. {revNumber}"
+                  values={{ revNumber: '999' }}
                 />
-              </EuiLink>
-            );
-          },
+              </EuiText>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        ),
+      },
+      {
+        field: 'description',
+        name: i18n.translate('xpack.ingestManager.agentConfigList.descriptionColumnTitle', {
+          defaultMessage: 'Description',
+        }),
+        render: (description: AgentConfig['description']) => (
+          <EuiTextColor color="subdued">{description}</EuiTextColor>
+        ),
+        width: '30%',
+      },
+      {
+        field: 'updated_on',
+        name: i18n.translate('xpack.ingestManager.agentConfigList.updatedOnColumnTitle', {
+          defaultMessage: 'Last updated on',
+        }),
+        render: (date: AgentConfig['updated_on']) => (
+          <FormattedDate value={date} year="numeric" month="long" day="2-digit" />
```

Comment:
change `month="long"` so that "March" reads as "Mar". I'm guessing `month="short"` but not 100% on props

Reply:
done